Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM drupal:8.9-apache
+FROM drupal:9.2-php7.4-apache
```

Comment:
I haven't looked at this project in awhile, but isn't the `Dockerfile9` file supposed to be for testing Drupal 9? eg. https://github.com/Lullabot/drupal9ci/blob/master/Dockerfile9#L1

Reply:
I think that @m4olivei is right: there was an automated build to create Drupal 9 images out of Dockerfile9. However, this seems to be now a premium feature so we will have to look for an alternative.

Reply:
I think that the easiest would be to set up a GitHub Action that builds and pushes the resulting image to [GitHub Packages](https://github.com/features/packages). Thoughts @fjgarlin ?

@mrdavidburns aren't you doing that already with other projects?

Reply:
We are not maintaining any custom GitHub packages at the moment. 
What we have done on other projects that are using GitHub Actions and Drupal 9 is just to install Ubuntu and then PHP 7.4. Most everything else happens through composer. 

Here's a snippet from `/.github/workflows/build.yml` in one of the sites we manage.

```
jobs:
  build:
    runs-on: ubuntu-latest
    timeout-minutes: 10
    steps:
      - name: Install PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '7.4'
          tools: pecl, composer:v2
          extensions: gd, intl, xsl
          coverage: none
```

Reply:
Gotcha, I was looking at something like this: https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

Reply:
Some quick context on the Dockerfiles:
* `Dockerfile` (the one this PR is tackling): this one is used in most places because it had PHP7.3 in it, and the right versions of other tools. **This file is already 100% compatible with Drupal 9**. This is the dockerfile used to build `juampynr/drupal8ci:latest`, used in all the different CI tools. See this: https://github.com/Lullabot/drupal9ci/search?q=drupal8ci. This dockerfile has an older version of node that is overriden in some of the files shown in the previous search.
* `Dockerfile9`: brings newer versions of the tools and has PHP8 in it. You can see it mostly in comments here: https://github.com/Lullabot/drupal9ci/search?q=juampynr%2Fdrupal9ci. If you are sure that your project supports PHP8, then you can pretty much switch all instances of `juampynr/drupal8ci` for `juampynr/drupal9ci` and you won't need to override node versions as explained in the comments.

So **both Dockerfiles prepare images that support Drupal 9**, and the biggest differences between them are the node (needed for Cypress tests) and PHP versions.

The code changes suggested on the PR seem pretty safe, but it'll be hard to test unless @juampynr builds the docker image and then we test with `juampynr/drupal8ci:latest`. Or as you mention @juampynr , maybe automate the image building and pushing it to Github packages.

Alternatively, you can build the docker image yourself, upload it somewhere publicly accessible and then replace all instances of `juampynr/drupal8ci` with `your-user/name-you-gave-to-the-image` in the CI tool that you're choosing. But again, the current image built out of this dockerfile is already D9 compatible, so maybe we don't need any changes at all.

I hope all this info helps a bit understanding the Dockerfiles in this project.